### PR TITLE
add public mobile war into mvn-client.yml

### PR DESCRIPTION
**Pull request by yzlucas**

### CODE REVIEW

File: .github/workflows/deploy-dev.yml [hunk 1]
```diff
@@ -108,6 +108,40 @@ jobs:
       AGOL_AREA_RESTRICTIONS: https://services6.arcgis.com/ubm4tcTYICKBpist/ArcGIS/rest/services/test_British_Columbia_Area_Restrictions/FeatureServer/0/
       AGOL_BANS_AND_PROHIBITIONS: https://services6.arcgis.com/ubm4tcTYICKBpist/ArcGIS/rest/services/test_British_Columbia_Bans_and_Prohibition_Areas/FeatureServer/0/
 
+      #public-mobile-war
+      bans.prohibitions.rss.location: config/bans-prohibitions-rss.xml
+      twitter.feed.profile.name: BCGovFireInfo
+      report.a.fire.phone: 1-800-663-5555
+      report.a.fire.cell: *5555
+      arc.gis .base.url: ${{ secrets.ARC.GIS.BASE.URL }}
```
Comment (vivid-cpreston) [inline .github/workflows/deploy-dev.yml:116]: typo here

File: .github/workflows/terragrunt-deploy.yml [hunk 2]
```diff
@@ -153,7 +153,112 @@ on:
...
+        type: string
+
+      report.a.fire.phone:
+        required: true
+        type: string
+      
+      report.a.fire.cell:
+        required: true
+        type: string
+      arc.gis .base.url:
```
Comment (vivid-cpreston) [inline .github/workflows/terragrunt-deploy.yml:172]: typo here

File: .github/workflows/deploy-dev.yml [hunk 3]
```diff
@@ -108,6 +108,40 @@ jobs:
       AGOL_AREA_RESTRICTIONS: https://services6.arcgis.com/ubm4tcTYICKBpist/ArcGIS/rest/services/test_British_Columbia_Area_Restrictions/FeatureServer/0/
       AGOL_BANS_AND_PROHIBITIONS: https://services6.arcgis.com/ubm4tcTYICKBpist/ArcGIS/rest/services/test_British_Columbia_Bans_and_Prohibition_Areas/FeatureServer/0/
 
+      #public-mobile-war
```
Comment (vivid-cpreston) [inline .github/workflows/deploy-dev.yml:111]: Input names can only contain alphanumeric containers, '-' or '_', not '.' characters. See https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_id

**Review by vivid-cpreston — COMMENTED** [review]
Structure looks good, input names will need modifying to match correct syntax,

**Comment by dhlevi** [comment]
Re-closing as we'll need to readdress with builds from nrids artifactory

**Pull request closed without merging**